index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+
+         <div class="imgplacesholder"></div>
+             <div>
+                 <h2 class="engin">
+                     Multi-Post Stories <br> Gain+Glory
+                 </h2>
+                 <ul class="eylem">
+                     <li class="irem">
+                         <button class="btn1">
+                             Ruby on rails
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn">
+                             css
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             JavScript
+                         </button>
+                     </li>
+                     <li class="irem">
+                         <button class="btn1">
+                             html
+                         </button>
+                     </li>
+                 </ul>
+                 <button class="btngreen">
+                     See Project
+                 </button>
+             </div>
+         </div>
+        </div>
+     </section>
+
+     <section id="about-me">
+        <h1>About me</h1>
+        <div class="img3"></div>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me. 
+        </p>
+        <div class="img4"></div>
+        <button class="btngreen">
+            Get My Resume
+        </button>
+     </section>
+
+     <section>
+      <div class="card">
+      <img class="logo-img" src="./images/Group-2.svg" alt="">
+      <h2>
+        Languages
+      </h2>
+      <ul class="eylem">
+        <li><button class="btn2">JavScript</button></li>
+        <li><button class="btn2">Ruby</button></li>
+        <li><button class="btn2">Html</button></li>
+        <li><button class="btn2">CSS</button></li>
+      </ul>
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038811649) from the previous reviewer. 

![Screenshot from 2022-12-04 01-21-44](https://user-images.githubusercontent.com/60083437/205458169-49448391-bb61-468e-9acb-d7516c3d2bcd.png)